monai/handlers/utils.py

```diff
@@ -228,3 +229,24 @@ def _compute_op(op: str, d: np.ndarray):
                     f.write(f"class{deli}{deli.join(ops)}\n")
                     for i, c in enumerate(np.transpose(v)):
                         f.write(f"{class_labels[i]}{deli}{deli.join([f'{_compute_op(k, c):.4f}' for k in ops])}\n")
+
+
+def from_dict(keys: Sequence[str]):
```

Comment:
from_dict is unclear to me, I think it should be `from_engine` or `from_engine_state`. I suppose this could also be used for `engine.state.batch`, why is it only described for the output? I think if this function doesn't have a good name and docstring, it'll be worse than a lambda function

Reply:
Hi @wyli ,

Thanks for your suggestions!
Yes, it can also be used with `engine.state.batch`, and we can even enhance it to extract keys from a list of dict after decollacting.
Let me try to update the name and doc-string.

Thanks.

Reply:
Updated in the latest commit.

Thanks.